statvar_imports/oecd/oecd_student_teacher_ratio/test_data/sample_input/.~lock.Student_Teacher_Ratio_OECD_test_data.csv#

```diff
@@ -0,0 +1 @@
+,swethakumari,swethakumari.c.googlers.com,26.07.2024 09:49,file:///usr/local/google/home/swethakumari/.config/libreoffice/4;
```

Comment:
Could you please remove the ~lock file for eg: statvar_imports/oecd/oecd_student_teacher_ratio/test_data/sample_input/.~lock.Student_Teacher_Ratio_OECD_test_data.csv#